.github/workflows/release.yaml

```
@@ -11,7 +11,7 @@ permissions:
   contents: read
 
 jobs:
-  check:
+  check-package-for-release:
```

We also trigger this `job` on `push` to the `main` branch.
```suggestion
  check-and-build-package:
```

---

Commit https://github.com/jenstroeger/python-package-template/pull/918/commits/2adc4d51b0cf429e08f8735255b452511930ec6d.